src/Modules/ContentBlocks/Domain/ContentBlock/ContentBlockType.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Domain\ContentBlock;
+
+use ForkCMS\Core\Domain\Form\EditorType;
+use ForkCMS\Core\Domain\Form\TitleType;
+use ForkCMS\Modules\Backend\Domain\User\User;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ContentBlockType extends AbstractType
+{
+    public function __construct(
+        private readonly Security $security
+    ) {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        /** @var ContentBlockDataTransferObject $data */
+        $data = $builder->getData();
+
+        /** @var User $user */
+        $user = $this->security->getUser();
+        if ($data->createdBy === null) {
+            $data->createdBy = $user;
+        }
+        $data->updatedBy = $user;
+
+        $builder->add(
+            'title',
+            TitleType::class
+        )->add(
+            'text',
+            EditorType::class,
+            [
+                'required' => true,
+                'label' => 'lbl.Content',
+            ]
+        );
+
+        /*$templates = $this->getPossibleTemplates($options['theme']);
```

Comment:
Wouldn't it be more logical to not commit commented code?

Reply:
I think I've used it once, but it was just to create a separate effect on displaying the content block. But that is something we should be able to solve with the editor right now.
If the ContentBlock entity implements the EntityWithSettingsTrait en somebody later wants to create a PR to add that again we can ask for usecases then. But I don't think we need it at the moment so it should be removed for now

Reply:
commented code removed: https://github.com/forkcms/forkcms/pull/3555/commits/0eee4916220942e1607775ea492552d6a300601a